### DISCUSSION TITLE
Error 429 Client Error

### DISCUSSION THREAD

**Discussion by Mariete124** [vzhd1701/csv2notion#3] [Q&A]
Hi again, I've been having this error a lot:
![image](https://user-images.githubusercontent.com/94454662/172153318-d7a86d0f-a565-4c88-bd24-8d394c12f80d.png)
I guees that is notion telling that I'm surpassing the amount of info per second, is there a way to bypass this?

**Comment by vzhd1701** [comment]
Do you use `--max-threads` option?

**Reply by Mariete124** [reply]
No, I should?

**Reply by vzhd1701** [reply]
Try setting it to 3 or less.

**Reply by Mariete124** [reply]
Yeaaa, I have to use 1 but It works!!! Thanks so much
